examples/automlbenchmark/README_automlbenchmark.md

```diff
@@ -125,7 +127,48 @@ python $PROJECT_PATH/runbenchmark.py AutoGluon test
    # python $PROJECT_PATH/runbenchmark.py AutoGluon:latest ag 1h8c -m aws -p 1500
    # python $PROJECT_PATH/runbenchmark.py AutoGluon:latest ag 4h8c -m aws -p 1500
    ```
-10. Aggregate AWS S3 results.
-    1. WIP
-11. Analyze aggregated results.
-    1. WIP
\ No newline at end of file
+   
+10. Advanced: Run distributed benchmark on your custom datasets on AWS
+
+    1. Prepare your datasets in s3 bucket with the following prefix: `arn:aws:s3:::automl-benchmark-*/ec2/*`. If you plan to have multiple folds, put them into separate files.
+
+    2. Create your custom benchmark yaml file under `automlbenchmark/resources/`, for example `custom_test.yaml`
+
+    3. Example benchmark yaml file:
+
+        ```yaml
+        - name: custom_dataset1
+          dataset:
+            train: 
+              - 's3://automl-benchmark-xxx/ec2/custom_datasets/train0.csv'
+              - 's3://automl-benchmark-xxx/ec2/custom_datasets/train1.csv'
+              - 's3://automl-benchmark-xxx/ec2/custom_datasets/train2.csv'
+            test: 
+              - 's3://automl-benchmark-xxx/ec2/custom_datasets/test0.csv'
+              - 's3://automl-benchmark-xxx/ec2/custom_datasets/test1.csv'
+              - 's3://automl-benchmark-xxx/ec2/custom_datasets/test2.csv'
+            target: class # This is the label to be predicted
+          folds: 3
+        
+        ```
+
+    4. Follow previous guide for running distributed benchmark on AWS. Remember to push and commit your changes.
+
+    5. To run the benchmark:
+
+       ```
+       mkdir -p benchmark_tmp
+       cd benchmark_tmp
+       # Run full benchmark suite (1h8c) on AWS, fully parallel, on AutoGluon custom branch / master with 'best_quality' preset
+       # Consider adding -f 0 to reduce compute by 10x (only train 1 fold instead of all 10), lowers confidence in results, but cheaper and faster.
+       # If you want to run on stable release of AutoGluon, remove :latest
+       # -p 1500 is just an example, change it to whatever suites your own benchmark
+       # python $PROJECT_PATH/runbenchmark.py AutoGluon_bestquality:latest custom_test 1h8c -m aws -p 1500
```

Comment:
nit: benchmark name consistent with example (`custom_dataset1`)

Reply:
This is actually correct because the benchmark is named by the yaml file(`custom_test.yaml`), the benchmark could contain multiple datasets. In this case, the one and only dataset is `custom_dataset1`

Reply:
ah, I see, good point. Thanks for clarifying!